rasa/core/policies/ensemble.py

```diff
@@ -113,15 +118,74 @@ def _check_priorities(self) -> None:
                     docs=DOCS_URL_POLICIES,
                 )
 
+    @staticmethod
+    def _split_ml_and_rule_trackers(
```

Comment:
I think it makes sense to rather do it inside the policies

Reply:
it will allow to create custom policies without subclassing rule policy directly

Reply:
We did that so we don't have to touch the other policies for now. We definitely should do that differently in the final implementation.

Reply:
let's add a TODO that this is temporary